website/client-old/js/controllers/settingsCtrl.js

```diff
@@ -2,14 +2,17 @@
 
 // Make user and settings available for everyone through root scope.
 habitrpg.controller('SettingsCtrl',
-  ['$scope', 'User', '$rootScope', '$http', 'ApiUrl', 'Guide', '$location', '$timeout', 'Content', 'Notification', 'Shared', '$compile',
-  function($scope, User, $rootScope, $http, ApiUrl, Guide, $location, $timeout, Content, Notification, Shared, $compile) {
+  ['$scope', 'User', '$rootScope', '$http', 'ApiUrl', 'Guide', '$location', '$timeout', 'Content', 'Notification', 'Shared', 'Social', '$compile',
+  function($scope, User, $rootScope, $http, ApiUrl, Guide, $location, $timeout, Content, Notification, Shared, Social, $compile) {
     var RELEASE_ANIMAL_TYPES = {
       pets: 'releasePets',
       mounts: 'releaseMounts',
       both: 'releaseBoth',
     };
 
+    var SOCIAL_AUTH_NETWORKS = window.habitrpgShared.constants.SUPPORTED_SOCIAL_NETWORKS
```

Comment:
This should be accessible as `Shared.constants.SUPPORTED_SOCIAL_NETWORKS`

Also, missing a `;`